tests/chart_tests/test_houston_configmap.py

```diff
@@ -4,859 +4,865 @@
+
+    def test_houston_configmap_with_loggingsidecar_readiness_probe(self):
+        """Validate the houston configmap with readiness probe configured."""
+        readiness_probe = {
+            "httpGet": {
+                "path": "/healthz",
+                "port": 8080,
+                "scheme": "HTTP",
+            },
+            "initialDelaySeconds": 15,
+            "timeoutSeconds": 30,
+            "periodSeconds": 5,
+            "successThreshold": 1,
+            "failureThreshold": 20,
+        }
+
+        docs = render_chart(
+            values={
+                "global": {
+                    "loggingSidecar": {
+                        "enabled": True,
+                        "name": "sidecar-log-test",
+                        "readinessProbe": readiness_probe,
+                    }
                 }
-            }
-        },
-        show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
-    )
+            },
+            show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
+        )
+
+        # Ensure the chart was rendered
+        assert len(docs) > 0
+
+        # Parse the rendered configmap
+        doc = docs[0]
+        prod_yaml = yaml.safe_load(doc["data"]["production.yaml"])
+
+        # Validate readinessProbe
+        assert "readinessProbe" in prod_yaml["deployments"]["loggingSidecar"]
+        assert prod_yaml["deployments"]["loggingSidecar"]["readinessProbe"] == readiness_probe
+
+    def test_houston_configmap_with_custom_airflow_ingress_annotation_with_authsidecar_disabled(self):
+        """Validate the houston configmap with custom airflow ingress annotation."""
+        docs = render_chart(
+            values={"global": {"extraAnnotations": {"route.openshift.io/termination": "passthrough"}}},
+            show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
+        )
+
+        # Ensure the chart was rendered
+        assert len(docs) == 1
+
+        # Parse the rendered configmap
+        doc = docs[0]
+        prod_yaml = yaml.safe_load(doc["data"]["production.yaml"])
+
+        # Validate extra ingress annotation
+        helm = prod_yaml["deployments"]["helm"]
+        assert "ingress" in helm
+        assert {"extraIngressAnnotations": {"route.openshift.io/termination": "passthrough"}} == helm["ingress"]
+
+    def test_houston_configmap_with_custom_airflow_ingress_annotation_disabled_with_authsidecar_disabled(self):
+        """Validate the houston configmap does not include airflow ingress annotation."""
+        docs = render_chart(
+            values={
+                "global": {"authSidecar": {"enabled": True}, "extraAnnotations": {"route.openshift.io/termination": "passthrough"}}
+            },
+            show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
+        )
 
-    # Ensure the chart was rendered
-    assert len(docs) > 0
+        # Ensure the chart was rendered
```

Comment:
We usually don't have comments like this in our tests, which seem unnecessary because what the code is doing is obvious. If we want to start adding comments that's ok, but IMHO these comments are not helpful. If others think they are helpful we can leave them. Personally I would remove them.